Flutter/pdf-viewer/add-remove-modify-annotations.md

```diff
@@ -17,7 +17,7 @@ This section will go through how to add annotations to a PDF document programmat
 
 ### Add annotations programmatically
 
-You can programmatically add a new annotation to the PDF document by creating an annotation instance and providing it as a parameter to the `addAnnotation` method of the `PdfViewerController` class. The following example shows how to create an instance of a highlight annotation and add it to the PDF document. Similarly, you can create and add other types of annotation.
+You can programmatically add a new annotation to the PDF document by creating an annotation instance and providing it as a parameter to the [addAnnotation](https://pub.dev/documentation/syncfusion_flutter_pdfviewer/latest/pdfviewer/PdfViewerController/addAnnotation.html) method of the `PdfViewerController` class. The following example shows how to create an instance of a highlight annotation and add it to the PDF document. Similarly, you can create and add other types of annotation.
```

Comment:
Apart from the text selection context menu customisation, you have done some additional things also. Please cover these also in the description

<img width="425" alt="image" src="https://github.com/syncfusion-content/flutter-docs/assets/65169077/57c3c7c0-2d41-41b4-92e8-d75fe8c29066">

Reply:
Edited the PR description